config/core/test-configs.yaml

```diff
@@ -1962,6 +1969,7 @@ test_configs:
       - kselftest-lkdtm
       - kselftest-rtc
       - kselftest-seccomp
+      - kselftest-vm
```

Comment:
OK: https://staging.kernelci.org/test/plan/id/62136a5c925f6bb55c4b3176/

It was run but results were not always parsed correctly. That seems to be more of an issue with the selftest to fix in the kernel tree (worked fine with volteer).